src/qt/addressbookpage.cpp

```diff
@@ -145,14 +150,14 @@ AddressBookPage::~AddressBookPage()
     delete ui;
 }
 
-void AddressBookPage::setModel(AddressTableModel *_model)
+void AddressBookPage::setModel(AddressTableModel *_model, const bool p2pkh_only)
```

Comment:
86be9e4a353f8c2ab35c3c1177e8ad63b8e50eab

nit, this could be renamed `AddressBookPage::setupModel(AddressTableModel* source_model, const bool p2pkh_only)`. Alternatively these could be passed in `AddressBookPage` constructor and this method would be private.

Reply:
This suggestion looks reasonable to me, @luke-jr will you get back to this?